### PR TITLE
modif css

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -67,8 +67,8 @@ function App() {
            */}
         </Routes>
       </main>
-      <Header />
       <Footer />
+      <Header />
```
Comment (LuigiMenez) [inline frontend/src/App.jsx:71]: Euh.... Why????

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

File: frontend/src/App.jsx (hunk 1 above)
Comment (RomainLF) [reply frontend/src/App.jsx:71]: to display menu burger in front of the footer in mobile version.

**Review by LuigiMenez — APPROVED** [review]
Ok

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**